### ISSUE TITLE
Search highlight test is unreadable in the night theme

### ISSUE THREAD

**Issue by mikebd** [thepracticaldev/dev.to#9074]
White on yellow does not work well.

![image](https://user-images.githubusercontent.com/112252/86374702-dda2de00-bc52-11ea-91bf-0b15980b1e33.png)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by nickytonline** [comment]
Did you want to take this issue on @mikebd?
